com/github/mashlol/MySQL.java

```diff
@@ -133,8 +168,11 @@ public void close() {
 		try {
 			con.close();
 		} catch (SQLException e) {
-			System.out.println("A bad call to mysql.close() occurred");	
-			e.printStackTrace();
+                            System.out.println("[StockMarket] " + "General Database error. Could not close SQL connection after use! Enable debug-mode for more info.");
+                            if (StockMarket.debugMode == true) {
+                                e.printStackTrace();
+                            }	
+                            dbstatus = false;
```

Comment:
That's pretty good, I was just trying to use Bukkit logging levels and cater for messages as well.
Up to you?

Sent from my iPhone

> On 2 Jan 2016, at 3:54 PM, Kevin Bedi <notifications@github.com> wrote:
> 
> In com/github/mashlol/MySQL.java:
> 
> > @@ -133,8 +168,11 @@ public void close() {
> >  		try {
> >  			con.close();
> >  		} catch (SQLException e) {
> > -			System.out.println("A bad call to mysql.close() occurred");	
> > -			e.printStackTrace();
> > +                            System.out.println("[StockMarket] " + "General Database error. Could not close SQL connection after use! Enable debug-mode for more info.");
> > +                            if (StockMarket.debugMode == true) {
> > +                                e.printStackTrace();
> > +                            }	
> > +                            dbstatus = false;
> I was thinking more like:
> 
> public void logError(String errorMessage, Exception e) {
>   System.out.println("[StockMarket] " + errorMessage);
>   if (StockMarket.debugMode) {
>     e.printStackTrace();
>   }
> }
> Not sure if we want to disable the plugin or not, maybe we do.
> 
> —
> Reply to this email directly or view it on GitHub.
>